### PR TITLE
Fix the ajax calls in the product page

### PR DESCRIPTION
**Pull request by antoin-m**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | The url parameters should not be concataned at the end of the ajax call urls since the token GET parameter is already present |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1581 http://forge.prestashop.com/browse/BOOM-1586 |
| How to test? | <ul><li>BO: Go to the product page</li><li>BO: You shouldn't get any "compromised" ajax call</li></ul> |

### CODE REVIEW

**Review by mickaelandrieu — APPROVED** [review]
Good job son!

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -480,8 +480,11 @@ var supplierCombinations = (function() {
       var suppliers = $('#form_step6_suppliers input[name="form[step6][suppliers][]"]:checked').map(function() {
         return $(this).val();
       }).get();
-      var url = collectionHolder.attr('data-url') + '/' + id_product + (suppliers.length > 0 ? '/' + suppliers.join('-') : '');
-
+      var url = collectionHolder.attr('data-url')
+        .replace(
+          /refresh-product-supplier-combination-form\/\d+\d+/,
```
Comment (thierrymarianne) [inline admin-dev/themes/default/js/bundle/product/form.js]: A single `\d+` would be enough.

**Review by thierrymarianne — CHANGES_REQUESTED** [review]

File: admin-dev/themes/default/js/bundle/product/form.js (hunk 1 above)
Comment (antoin-m) [inline admin-dev/themes/default/js/bundle/product/form.js]: Oops! I missed a `/` between both `\d+` !

**Comment by dSkrbic** [comment]
Hello,

Q&A test **NOK**

**Comment by mickaelandrieu** [comment]
Q&A tests OK.

**Pull request merged**